process-dataset/data-validation/schema.js

```diff
@@ -159,7 +163,10 @@ const featureDefSchema = {
             "description": "For discrete features, display items for each value",
             "type": "object",
         },
-
+        "exclude": {
+            "description": "Parts of the app to exclude this feature from. Options are 'plot', 'colorBy', 'groupBy'",
+            "type": "array",
+        }
```

Comment:
I am not a big fan of this.  This is putting app functionality into a part of the data where it doesn't need to be.  I feel that the raw feature defs should not know about "group by" or have references to it.  Would you consider putting the exclude list in dataset.json where the default group-by feature is named, and the lists of feature display order, etc is described?

Reply:
Oh, I thought this was what we had decided on when we discussed this. Yeah I can do that. 
we could make a 
```
colorBy/groupBy/xAxis/yAxis: {
default: "",
exclude: [],
}
```
for each of these things. What do you think?

Reply:
yes that's more like what I was picturing.  and in fact you could put an  "include" list as well, which would allow users to define the order right there (and override the exclude list maybe...)

Reply:
can remove?